crates/edr_eth/src/transaction/signed/eip155.rs

```diff
@@ -23,17 +23,13 @@ pub struct Eip155 {
     pub kind: TxKind,
     pub value: U256,
     pub input: Bytes,
-    pub signature: Signature,
+    #[cfg_attr(feature = "serde", serde(flatten))]
+    pub signature: signature::Fakeable<signature::Ecdsa>,
```

Comment:
I'm wondering why `signature::Fakeable<signature::Ecdsa>` and not just `signature::Ecdsa`?

Reply:
On a second thought I think I understand now, but it's a bit weird to have a fakeable field in what is an RPC type (and transactions from RPC should definitely not be fake). Feels like we need two signed transaction types: one unverified (which comes from user input or we fetch it from RPC) and one verified where we can bypass the verification with fake signatures.

Reply:
Right, thanks, I got confused by the serde implementation. Thought the internal type didn't have it.